app/packages/relay/src/queries/countValues.ts

```diff
@@ -1,11 +1,17 @@
 import { graphql } from "react-relay";
 
 export default graphql`
-  query countValuesQuery($dataset: String!, $view: BSONArray!, $path: String!) {
+  query countValuesQuery(
+    $dataset: String!
+    $view: BSONArray!
+    $path: String!
```

Comment:
Probably a dumb question, but are the paths from the extended view not already calculated/handled)? I recall seeing some code in `core/aggregations.py` and `aggregations.ts` that seem to be constantly requerying aggregations, but I haven't looked into the exact parameters of the requests.
 
cc @benjaminpkane

Reply:
Previously aggregations were grouped into a single request. But now they are issued separately/explicitly as a parameter